docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -3,10 +3,16 @@
 
 [role="_abstract"]
 
-A devfile describes and defines a cloud native development environment.
+Devfiles are YAML files that describe and define a developer's workspaces or cloud-native-development-environments. Devfiles also describe best practices for end-to-end application development while still being customized or extended to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating what was previously done manually.
```

Comment:
```suggestion
Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development while providing a way for being customized or extended to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating what was previously done manually.
```